--- a/nymya_event_class_syscall_exit.c
+++ b/nymya_event_class_syscall_exit.c
@@ -27,11 +27,8 @@
         return 0;
     }
-EXPORT_SYMBOL_GPL(nymya_event_class_syscall_exit);
-
-
 
     // Export the symbol so other kernel modules/code can call it directly.
-    EXPORT_SYMBOL_GPL(nymya_event_class_syscall_exit);
+    EXPORT_SYMBOL_GPL(event_class_syscall_exit);
 
 #else // Userland implementation
 
